### PR TITLE
Add Ukraine statement and call for donations

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Review by bwplotka — APPROVED** [review]
Thanks, Julius!

**Review by metalmatze — APPROVED** [review]
💯

**Review by SuperQ — APPROVED** [review]
LGTM

File: content/index.html [hunk 1]
```diff
@@ -12,6 +12,14 @@ <h1>From metrics to insight</h1>
   </div>
 </div>
 
+<div class="newsbar" style="font-weight: normal; background-color: #222; color: #ffbc24;">
+  The Prometheus Team strongly condemns Russia's illegal attack war on Ukraine.
```
Comment (SuperQ) [inline content/index.html:16]:
```suggestion
  The Prometheus Team strongly condemns Russia's illegal invasion of Ukraine.
```

**Comment by fstab** [comment]
:heart: thanks @juliusv!

**Review by roidelapluie — COMMENTED** [review]
LGTM!

**Comment by RichiH** [comment]
LGTM

**Comment by RichiH** [comment]
See https://github.com/prometheus/docs/pull/2105 for more precise and fair wording.

**Comment by juliusv** [comment]
Was considering this initially as well, but it's Putin's entire political
circle, plus it still seems like a majority of the country are still
supporters (due to propaganda, but still).

**Comment by RichiH** [comment]
100% of my sample set of Russians living both inside and outside of Russia is against it.

I would question any data source claiming certainty in a question in which one of two possible answers will put you in jail for up to 15 years and/or conscripted and forced to fight in the front-line.

All in all, I would rather err on the side of compassion both ways, in particular because the proposed change does not weaken the message in any way.

**Comment by juliusv** [comment]
Yeah I wouldn't say "Russians'", but "Russia's" seems fair to me at this
point. But I'm also fine to change it if you feel strongly.

**Comment by RichiH** [comment]
Thank you, done.

**Comment by juliusv** [comment]
I agree with https://twitter.com/Afelia/status/1504686280864641027. Just saying "Putin's" removes responsibility from all the war criminals and the people who cheer them on.

**Comment by RichiH** [comment]
I agree with the sentiment, but have not found a way to transport the
required nuance in a way that fits into a 2-sentence statement.

If consensus goes toward "Russia" I won't argue the point strongly,
though I wonder if we transport a substantially better message with
any such change.

Is there a nuanced consideration we can link to from our statement?

**Comment by roidelapluie** [comment]
I prefer "Russia's" as well.

**Comment by juliusv** [comment]
PR to change it back to "Russia's": https://github.com/prometheus/docs/pull/2110

Maybe we should even make that whole statement stronger now, "invasion" sounds so harmless judging by the genocide that has been unfolding on the ground there.

**Pull request merged**